frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/EditActions/ActionsEditor/PredefinedActions/PredefinedActions.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { StudioNativeSelect } from '@studio/components';
+import { useActionHandler } from '../hooks/useOnActionChange';
+import { useBpmnContext } from '../../../../../../contexts/BpmnContext';
+import { getPredefinedActions, isActionAvailable } from '../../../../../../utils/processActions';
+import { type Action, BpmnActionModeler } from '../../../../../../utils/bpmn/BpmnActionModeler';
+
+type PredefinedActionsProps = {
+  actionElement: Action;
+};
+export const PredefinedActions = ({
+  actionElement,
+}: PredefinedActionsProps): React.ReactElement => {
+  const { t } = useTranslation();
+  const { bpmnDetails } = useBpmnContext();
+  const bpmnActionModeler = new BpmnActionModeler(bpmnDetails.element);
+  const { handleOnActionChange } = useActionHandler(actionElement);
+
+  const actions = bpmnActionModeler.actionElements?.action || [];
+  const availablePredefinedActions = getPredefinedActions(bpmnDetails.taskType);
+
+  const shouldDisableAction = (action: string): boolean => {
+    return !isActionAvailable(action, actions) && !(action === actionElement.action);
+  };
+
+  const isPredefinedAction = (action: string): boolean => {
+    return availablePredefinedActions.includes(action);
+  };
+
+  return (
+    <StudioNativeSelect
+      size='small'
+      label={t('process_editor.configuration_panel_actions_action_selector_label')}
+      onChange={handleOnActionChange}
+      defaultValue={isPredefinedAction(actionElement.action) ? actionElement.action : ' '}
+    >
+      <option disabled value=' '></option>
```

Comment:
Is it possible to do like this? 🤔 
```suggestion
      <option disabled value=' ' />
```

Reply:
Yes, it is! Changed it :)